internal/cmd/server/metrics.go

```diff
@@ -29,7 +29,7 @@ var metricTypeStrings = []string{
 var MetricsCmd = base.Cmd{
 	BaseCobraCommand: func(client hcapi2.Client) *cobra.Command {
 		cmd := &cobra.Command{
-			Use:                   "metrics [FLAGS] SERVER",
+			Use:                   "metrics [options] <server>",
```

Comment:
`--type` is required.

ow are required flags with enum values that can be specified multiple times defined in docopt? `(--type <cpu,disk,network>)...` perhaps?

Reply:
I don't think there is a standardized approach to this. I like your suggestion. Maybe we can use the regular convention for enums inside the angled brackets (`cpu|disk|network`) to make it clear that it's an enum. Then `...` would still work: `(--type <cpu|disk|network>)...` <-> `[(--type cpu) (--type disk) (--type network)]`

Reply:
Yes, I meant to point out that this is what the syntax would expand to ("proving" that it is still valid). Of course we wouldn't want this for the actual usage string.